.github/CONTRIBUTING.md

```diff
@@ -67,6 +68,21 @@ Enter password for new role: peertube
 # sudo -u postgres createdb -O peertube peertube_dev
 ```
 
+You also need to create a system user for Peertube:
+
+```
+$ sudo mkdir -p /var/www/
+$ sudo useradd -m -d /var/www/peertube -s /bin/bash -p peertube peertube
+$ sudo passwd peertube
+```
+
+Then enable extensions PeerTube needs:
+
+```
+$ sudo -u postgres psql -c "CREATE EXTENSION pg_trgm;" peertube_dev
+$ sudo -u postgres psql -c "CREATE EXTENSION unaccent;" peertube_dev
+```
+
 In dev mode, administrator username is **root** and password is **test**.
```

Comment:
You should remove stars cause if like me you read this doc in a basic text editor you will be confused about the password.